drivers/display/display_ili9341.c

```diff
@@ -134,5 +135,10 @@ int ili9341_regs_init(const struct device *dev)
 		return r;
 	}
 
+	r = ili9xxx_transmit(dev, ILI9XXX_DISPON, NULL, 0U);
+	if (r < 0) {
+		return r;
+	}
```

Comment:
I don't think this is the correct fix, `ILI9XXX_DISPON` is normally sent with `ili9xxx_display_blanking_off`. The problem with this specific board is that the display needs that command to be sent for both the LTDC AND the ILI9xxx driver.

Since this is a board specific quirk I think that the correct fix is to have a board specific file grabbing the `ili9341` display device (with a nodelabel or something) and calling the API unblank command at the right time.

Reply:
Okey, thanks! Will make the necessary modifs.